tests/consumers/test_message_processors.py

```diff
@@ -10,13 +11,17 @@
 from snuba.processor import InsertBatch
 
 
-def test_message_processors() -> None:
+@pytest.mark.parametrize(
+    "topic,processor_name",
+    [
+        ("snuba-queries", "QuerylogProcessor"),
+        ("profile-metadata", "ProfilesMessageProcessor"),
```

Comment:
should it be `processed-profiles`? Needs to match the (default) topic name otherwise the examples won't be loaded